packages-content-model/roosterjs-content-model-editor/lib/publicApi/utils/paste.ts

```diff
@@ -135,7 +139,7 @@ function triggerPluginEventAndCreatePasteFragment(
     pasteAsText: boolean,
     pasteAsImage: boolean,
     eventData: ContentModelBeforePasteEventData
-): { pluginEvent: ContentModelBeforePasteEvent; fragment: DocumentFragment } {
+): ContentModelBeforePasteEvent | ContentModelBeforePasteEventData {
```

Comment:
Since `ContentModelBeforePasteEvent` inherits from `ContentModelBeforePasteEventData` and we didn't use anything in `ContentModelBeforePasteEvent` but not in `ContentModelBeforePasteEventData`, we can just return  `ContentModelBeforePasteEventData`

Reply:
Good catch. fixed in latest commit.